### PR TITLE
Add improvements.

### PR DESCRIPTION
**Pull request by dewniMW**

### CODE REVIEW

File: en/docs/references/adaptive-authentication-js-api-reference.md [hunk 1]
```diff
@@ -239,7 +235,7 @@ following parameters.
 <tbody>
 <tr class="odd">
 <td>url</td>
-<td>The URL of the error page that the user is redirected to. If the value is null, the user is redirected by default to the ' <strong>retry.do'</strong> error page.<br />
+<td>The URL of the error page that the user is redirected to. If the value is null, the user is redirected by default to the <strong>retry.do</strong> error page.<br />
```
Comment (gomathyk) [inline en/docs/references/adaptive-authentication-js-api-reference.md:238]:
```suggestion
<td>This is the URL of the error page that the user is redirected to. If the value is null, the user is redirected by default to the <strong>retry.do</strong> error page.<br />
```

File: en/docs/references/adaptive-authentication-js-api-reference.md [hunk 2]
```diff
@@ -366,19 +362,19 @@ the following parameters.
...
 <tr class="even">
-<td>value</td>
-<td>Value of the cookie.</td>
+<td>name</td>
+<td>Name of the cookie.</td>
 </tr>
 <tr class="odd">
 <td>properties</td>
-<td><p>A map that may contain optional attributes of the cookie ‘ <code>                             de                           </code> <code>                             crypt                           </code> <strong>validateSignature</strong></p>
+<td>A map that may contain optional attributes of the cookie <code>decrypt</code> and <code>validateSignature</code>
```
Comment (gomathyk) [inline en/docs/references/adaptive-authentication-js-api-reference.md:374]: Missing fullstop

File: en/docs/references/adaptive-authentication-js-api-reference.md [hunk 3]
```diff
@@ -607,35 +606,35 @@ step number.
...
 -   `          application.appId         ` :  This is the id of the application.
     
-##### userAgent Object
+#### userAgent object
 
 -   `          userAgent.raw         ` :  This is the raw userAgent string.
--   `          userAgent.browser         ` :  This is the Web Browser property that is extracted from the raw userAgent string.
--   `          userAgent.platform         ` :  This is the Operating System property that is extracted from the raw userAgent string.
--   `          userAgent.device         ` :  This is the Device property that is extracted from the raw userAgent string.
+-   `          userAgent.browser         ` :  This is the web browser property that is extracted from the raw userAgent string.
```
Comment (gomathyk) [inline en/docs/references/adaptive-authentication-js-api-reference.md:637]: Shall we give placeholders like userAgent as codes? 
The same applies to 'true' and 'false'. 

Please check wherever applicable.

**Review by gomathyk — APPROVED** [review]

File: en/docs/references/adaptive-authentication-js-api-reference.md (hunk 3 above)
Comment (dewniMW) [reply en/docs/references/adaptive-authentication-js-api-reference.md:637]: fixed in https://github.com/wso2/docs-is/pull/2617

File: en/docs/references/adaptive-authentication-js-api-reference.md (hunk 2 above)
Comment (dewniMW) [reply en/docs/references/adaptive-authentication-js-api-reference.md:374]: fixed in https://github.com/wso2/docs-is/pull/2617

**Pull request merged**